fastlane/Fastfile

```diff
@@ -14,14 +14,15 @@
 # update_fastlane
 
 gradle_file_path = "library.gradle"
-files_with_version_number = [
-  gradle_file_path,
-  './common/src/main/java/com/revenuecat/purchases/common/Config.kt',
-  './.circleci/config.yml',
-  './docs/index.html',
-  './gradle.properties',
-  './.version'
-]
+files_with_version_number = {
+  gradle_file_path => ['versionName "{x}"'],
+  './common/src/main/java/com/revenuecat/purchases/common/Config.kt' => ['frameworkVersion = "{x}"'],
+  './.circleci/config.yml' => ['aws s3 sync ~/project/docs/{x} s3://purchases-docs/android/{x} --delete'],
```

Comment:
I guess this could be done as 2 different matches... But this also works. I think we didn't have a test for a similar replacement pattern? (with more than 1 match locations?)

Reply:
We don't have a test for that, I will take notes and add it after my OOO